### ISSUE TITLE
make doc site toggle between options and composition api just like vue site

### ISSUE THREAD

**Issue by laoshaw** [posva/pinia#1265]
### What problem is this solving

it's hard to follow pinia's document due to the mixing of options-api and composition-api, also the composition api is not vue3.2 ready(script-setup)

### Proposed solution

make the doc switchable between options and composition mode

### Describe alternatives you've considered

the same way vue's doc site

**Comment by posva** [comment]
~~Added to #829 but unsure if this is feasible/worth since it's not exactly like in Vue docs.~~

It was already there.
